spec/setup_acceptance_node.pp

```diff
@@ -3,11 +3,18 @@
 }
 
 # Needed for idempotency when SELinux is enabled
-if $foreman::repo::configure_scl_repo {
-  package { 'rh-redis5-redis':
-    ensure  => installed,
+if $facts['os']['selinux']['enabled'] {
+  package { 'foreman-selinux':
+    ensure  => latest,
     require => Class['foreman::repo'],
   }
+
+  if $foreman::repo::configure_scl_repo {
+    package { 'rh-redis5-redis':
+      ensure  => installed,
+      require => Class['foreman::repo'],
+    }
+  }
```

Comment:
This whole block is indented too far I think. And you'll have a trailing `}` on line 18.

Reply:
It was my intention to have the `rh-redis5-redis` package inside `if $facts['os']['selinux']['enabled'] { ... }` because of the existing comment on L#5: `# Needed for idempotency when SELinux is enabled` (which now conveniently applies to both package resources)

IMO it was an oversight that this wasn't already conditioned on selinux being enabled.

I thought about making the change in a separate commit but it seemed too minor to warrant it. That would have made this easier to review but probably the git log clutter is not worth it when this only affects tests.

What do you think?

Reply:
That works. Can you rebase on latest master branch to ensure tests pass?